troposphere/autoscaling.py

```diff
@@ -56,7 +56,7 @@ def JSONrepr(self):
         return self.tags
 
 
-class NotificationConfiguration(AWSProperty):
```

Comment:
Thank you for implementing these changes. I was wondering if NotificationConfiguration still worked and whether to leave it as a backwards compatible interface. Thoughts?